client/src/app/(modules)/practices/(main)/page.tsx

```diff
@@ -105,7 +106,12 @@ export default function PracticesModule() {
           )}
         </Button>
       </div>
-      <PracticeList {...practices} />
+      <div className="border-t border-dashed border-t-gray-300 pt-6 text-sm text-gray-500">
+        Showing {practicesCount} practice(s).
```

Comment:
Can't we add the "s" depending on the count? Or is this a requirement?

Reply:
I did this to match what Tiago did on the Network module but I don't mind changing both so we only display the “s” when needed 😉.

Reply:
I think that will look much better. Thanks!